README.md

```diff
@@ -256,6 +256,22 @@ To get in touch with the developers,
 [join us on Slack](https://join.slack.com/t/trixi-framework/shared_invite/zt-sgkc6ppw-6OXJqZAD5SPjBYqLd8MU~g)
 or [create an issue](https://github.com/trixi-framework/Trixi.jl/issues/new).
 
+## Institutional Support
+Trixi.jl relies on institutions that provide infrastructure for coding and testing, as well as encourage students, post-docs and scientists to contribute code. Institutes and groups that supported us in the past and present are listed below.
+
+<p align="left">
+  <img style="float:left;height:30px;margin-right:10px" 
+    src="https://github.com/torrilhon/Trixi.jl/assets/17232977/e9432f4c-995f-434e-b393-d15efe68abf1">
+  <a href="https://www.mi.uni-koeln.de/NumSim/">Numerical Simulation Group, University of Cologne</a>
+</p>
+
+<p align="left">
```

Comment:
Here are some additional institutes that have supported us significantly (in chronological order as far as I remember)

```suggestion
</p>

- [Applied Mathematics, Linköping University, Sweden](https://liu.se/en/organisation/liu/mai/tima)
- [Applied Mathematics, University of Münster, Germany](https://www.uni-muenster.de/AMM/en/)
- [Computational and Applied Mathematics, Rice University, USA](https://cmor.rice.edu)
- [Applied Mathematics, University of Hamburg, Germany](https://www.math.uni-hamburg.de/en/forschung/bereiche/am.html)

<p align="left">
```

We should add their logos etc. as well, of course

Reply:
Shouln't Uni Mainz be included in this list?

Reply:
It is - but I moved to Mainz after people from Aachen started contributing, I think 🤔 That's why I have added Mainz below.

Reply:
Whoops, I missed that, my bad.